app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java

```diff
@@ -38,4 +31,12 @@ public void showDialog(ListenerDialogFragment listenerDialogFragment) {
             dialogFragment.setCancelable(false);
         }
     }
+
+    @Override
+    public void showPopUp(int spaces) {
+        Context context = getContext();
+        if (context != null) {
+            Snackbar.make(binding.getRoot(), context.getResources().getString(R.string.snack_bar_button_click_message_main_activity, String.valueOf(spaces)), Snackbar.LENGTH_SHORT).show();
```

Comment:
instead of String.valueOf(spaces)) you can use spaces.toString() I think

Reply:
based on what we talked about this comment I will let it like it is